tests/yam/agama/playwright_test.pm

```diff
@@ -0,0 +1,32 @@
+## Copyright 2023 SUSE LLC
```

Comment:
let's use the same name for yaml than test module hiding the technology as well.

Reply:
updated